pkg/backup/download.go

```diff
@@ -257,15 +258,15 @@ func (b *Backuper) downloadTableData(remoteBackup metadata.BackupMetadata, table
 
 	if remoteBackup.DataFormat != "directory" {
 		capacity := 0
-		for disk := range table.Files {
-			capacity += len(table.Files[disk])
+		for disk := range table.DiskToFiles {
+			capacity += len(table.DiskToFiles[disk])
 		}
-		apexLog.Debugf("start downloadTableData %s.%s with concurrency=%d len(table.Files[...])=%d", table.Database, table.Table, b.cfg.General.DownloadConcurrency, capacity)
+		apexLog.Debugf("start downloadTableData %s.%s with concurrency=%d len(table.DiskToFiles[...])=%d", table.Database, table.Table, b.cfg.General.DownloadConcurrency, capacity)
 
-		for disk := range table.Files {
-			diskPath := b.DiskMap[disk]
-			tableLocalDir := path.Join(diskPath, "backup", remoteBackup.BackupName, "shadow", dbAndTableDir, disk)
-			for _, archiveFile := range table.Files[disk] {
+		for disk := range table.DiskToFiles {
+			backupPath := b.DiskToPathMap[disk]
```

Comment:
It's not a whole backup path
when you have multiple disk volumes in your storage policy
`diskPath` here more mental name
look to https://clickhouse.com/docs/en/engines/table-engines/mergetree-family/mergetree/#terms